README.md

```diff
@@ -1,20 +1,100 @@
 # ConduitMQTT
 
-WIP
+An MQTT adapter for [Conduit](https://github.com/conduitframework/conduit).
+
+## Installation
+
+This package can be installed as:
+
+  1. Add `conduit_mqtt` to your list of dependencies in `mix.exs`:
+
+    ```elixir
+    def deps do
+      [{:conduit_mqtt, "~> 0.1.0"}]
+    end
+    ```
+
+  2. Ensure `conduit_mqtt` is started before your application:
+
+    ```elixir
+    def application do
+      [applications: [:conduit_mqtt]]
+    end
+    ```
+
+## Configuring the Adapter
+
+```elixir
+# config/config.exs
+
+config :my_app, MyApp.Broker,
+  adapter: ConduitMQTT, connection_opts: [server: {Tortoise.Transport.Tcp, host: 'localhost', port: 1883}]  #TODO IS THIS CORRECT WAY TO GET INTO ADAPTER OPTS?
+
+# Stop lager redirecting :error_logger messages
+config :lager, :error_logger_redirect, false
+
+# Stop lager removing Logger's :error_logger handler
+config :lager, :error_logger_whitelist, [Logger.ErrorHandler]
```

Comment:
These two lager things should be unnecessary unless tortoise is bringing it in. The rabbit stuff does, which is why it was there.